WildFly_26.1.0.Final/jakarta-web-profile.adoc

```diff
@@ -0,0 +1,244 @@
+
+Test name: jsf
+[javatest.batch] Completed running 5526 tests.
+[javatest.batch] Number of Tests Passed      = 5526
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jsonb
+[javatest.batch] Completed running 532 tests.
+[javatest.batch] Number of Tests Passed      = 532
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jsonp
+[javatest.batch] Completed running 372 tests.
+[javatest.batch] Number of Tests Passed      = 372
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jsp
+[javatest.batch] Completed running 721 tests.
+[javatest.batch] Number of Tests Passed      = 721
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jstl
+[javatest.batch] Completed running 541 tests.
+[javatest.batch] Number of Tests Passed      = 541
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jta
+[javatest.batch] Completed running 154 tests.
+[javatest.batch] Number of Tests Passed      = 154
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: securityapi
+[javatest.batch] Completed running 83 tests.
+[javatest.batch] Number of Tests Passed      = 83
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: servlet
+[javatest.batch] Completed running 1643 tests.
+[javatest.batch] Number of Tests Passed      = 1643
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: signaturetest_javaee
+[javatest.batch] Completed running 2 tests.
+[javatest.batch] Number of Tests Passed      = 2
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: websocket
+[javatest.batch] Completed running 745 tests.
+[javatest.batch] Number of Tests Passed      = 745
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+----
+
+=== Additional standalone TCK Test results
+Standalone test results:
+
+Jakarta Dependency Injection 1.0 TCK
+
+Download URL & SHA-256:
+https://download.eclipse.org/jakartaee/dependency-injection/1.0/jakarta.inject-tck-1.0-bin.zip[jakarta.inject-tck-1.0-bin.zip, b679bac9b1057df894753892a880ba6ade530607dd811157106ed767aa26481f ]
+
+TCK result summary:
+----
+Tests run: 50, Failures: 0, Errors: 0, Skipped: 0
+----
+
+Jakarta Contexts and Dependency Injection 2.0 TCK
+
+Download URL & SHA-256
+https://download.eclipse.org/jakartaee/cdi/2.0/cdi-tck-2.0.6-dist.zip[cdi-tck-2.0.6-dist.zip, 75e969a7a3b3c77332154a2008309aad821a923d8684139242048a7640762808 ]
+
+TCK result summary:
+----
+Tests run: 1807, Failures: 0, Errors: 0, Skipped: 0
```

Comment:
@scottmarlow I believe the next line is missing. Should be:

----

I was looking at another one of these earlier today and the formatting toward the bottom looked off; I suspect this is why.

Reply:
Fixed!